resources/views/bread/edit-add.blade.php

```diff
@@ -63,11 +63,11 @@
                                 <div class="form-group">
                                     <label for="name">{{ $row->display_name }}</label>
 
-                                    @includeIf("voyager::formfields.$row->type")
+                                    {!! app('voyager')->formField($row, $dataType, $dataTypeContent) !!}
```

Comment:
We could use facade yea. Not sure what is best with performance.
Basicly it does the same.